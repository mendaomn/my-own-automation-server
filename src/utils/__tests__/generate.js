const uniq = require('lodash.uniq')

const generate = require('../generate')

test('should generate different ids every time', () => {
  let uuids = []

  for (let i = 0; i < 100; i++) {
    uuids.push(generate.id())
  }

  expect(uuids).toEqual(uniq(uuids))
});