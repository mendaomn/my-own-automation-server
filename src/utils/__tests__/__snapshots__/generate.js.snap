// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should generate makefiles 1`] = `
Object {
  "data": Array [
    46,
    80,
    72,
    79,
    78,
    89,
    58,
    32,
    97,
    108,
    108,
    32,
    116,
    101,
    115,
    116,
    32,
    98,
    117,
    105,
    108,
    100,
    10,
    10,
    97,
    108,
    108,
    58,
    32,
    116,
    101,
    115,
    116,
    32,
    98,
    117,
    105,
    108,
    100,
    10,
    10,
    116,
    101,
    115,
    116,
    58,
    10,
    9,
    101,
    120,
    105,
    116,
    32,
    48,
    10,
    10,
    98,
    117,
    105,
    108,
    100,
    58,
    10,
    9,
    101,
    120,
    105,
    116,
    32,
    48,
    10,
  ],
  "type": "Buffer",
}
`;

exports[`should generate makefiles 2`] = `
Object {
  "data": Array [
    46,
    80,
    72,
    79,
    78,
    89,
    58,
    32,
    97,
    108,
    108,
    32,
    116,
    101,
    115,
    116,
    32,
    98,
    117,
    105,
    108,
    100,
    10,
    10,
    97,
    108,
    108,
    58,
    32,
    116,
    101,
    115,
    116,
    32,
    98,
    117,
    105,
    108,
    100,
    10,
    10,
    116,
    101,
    115,
    116,
    58,
    10,
    9,
    101,
    120,
    105,
    116,
    32,
    49,
    10,
    10,
    98,
    117,
    105,
    108,
    100,
    58,
    10,
    9,
    101,
    120,
    105,
    116,
    32,
    49,
    10,
  ],
  "type": "Buffer",
}
`;
